src/docfx/build/manifest/ManifestBuilder.cs

```diff
@@ -121,5 +129,14 @@ public bool TryAdd(Document file, FileManifest manifest)
                 _manifest.Values.OrderBy(item => item.SourcePath).ToArray(),
                 _manifest.Keys.OrderBy(item => item.FilePath).ToList());
         }
+
+        private bool CheckMonikerConflict(List<string> existingMonikers, List<string> currentMonikers)
+        {
+            if (existingMonikers.Intersect(currentMonikers).Count() > 0 || existingMonikers.Concat(currentMonikers).Count() == 0)
```

Comment:
>existingMonikers.Concat(currentMonikers).Count() == 0 [](start = 75, length = 53)

Is this check for 2 duplicate outputs without monikers? Seems it will miss error when first adding some monikers to existingMonikers, then adding 2 non-version outputs with same outputUrl.